src/Game.coffee

```diff
@@ -243,11 +244,20 @@ class Game
     @sendMessage redraw, @, @
     @score += POINTS_PER_COIN
     @updateScore()
+    # perfectly counts and updates remaining coins on the board
+    #@board.revealPitTiles()
+    console.log("Coins Remaining: #{@board.coinsRemaining()}")
     if @board.coinsRemaining() <= 0
-      @board.revealAll()
-      @sendMessage redraw, @, @
-      # TODO - [rmarks] reset the board
+      @board.revealPitTiles()
+      @timer.pauseTimer = true
+      @board.pauseCollectCoins = true
```

Comment:
as mentioned before, the logic for pausing the input should probably be refactored to be at the Game level rather than the board level